src/main/java/camp/woowak/lab/cart/service/CartService.java

```diff
@@ -0,0 +1,40 @@
+package camp.woowak.lab.cart.service;
+
+import org.springframework.stereotype.Service;
+
+import camp.woowak.lab.cart.domain.Cart;
+import camp.woowak.lab.cart.exception.MenuNotFoundException;
+import camp.woowak.lab.cart.repository.CartRepository;
+import camp.woowak.lab.cart.service.command.AddCartCommand;
+import camp.woowak.lab.menu.domain.Menu;
+import camp.woowak.lab.menu.repository.MenuRepository;
+
+@Service
+public class CartService {
+	private final CartRepository cartRepository;
+	private final MenuRepository menuRepository;
+
+	public CartService(CartRepository cartRepository, MenuRepository menuRepository) {
+		this.cartRepository = cartRepository;
+		this.menuRepository = menuRepository;
+	}
+
+	/**
+	 * @throws MenuNotFoundException                                  메뉴가 존재하지 않는 경우
+	 * @throws camp.woowak.lab.cart.exception.OtherStoreMenuException 다른 가게의 메뉴를 담은 경우 도메인에서 발생
+	 * @throws camp.woowak.lab.cart.exception.StoreNotOpenException   해당 가게가 열려있지 않은 경우 발생
+	 */
+	public void addMenu(AddCartCommand command) {
+		Cart customerCart = getCart(command.customerId());
+
+		Menu menu = menuRepository.findByIdWithStore(command.menuId())
+			.orElseThrow(() -> new MenuNotFoundException(command.menuId() + " not found"));
+
+		customerCart.addMenu(menu);
```

Comment:
Cart는 Jpa가 관리하는 Entity가 아니기 때문에 cartRepository에 다시 한번 저장해주어야 하겠습니다.

Reply:
```suggestion
		customerCart.addMenu(menu);
```
이 부분을 한 뒤, customerCart를 다시 저장하라는 말씀이신가요?

Reply:
네네 맞습니다.

Reply:
아하 맞네요 다시 저장해줘야겠네요 감사합니다! :)